sci-libs/opencascade/opencascade-0.18.3.recipe

```diff
@@ -0,0 +1,200 @@
+	devel:libTKPLCAF$secondaryArchSuffix = $libVersionCompat
+	devel:libTKPrim$secondaryArchSuffix = $libVersionCompat
+	devel:libTKPShape$secondaryArchSuffix = $libVersionCompat
+	devel:libTKShapeSchema$secondaryArchSuffix = $libVersionCompat
+	devel:libTKShHealing$secondaryArchSuffix = $libVersionCompat
+	devel:libTKStdLSchema$secondaryArchSuffix = $libVersionCompat
+	devel:libTKSTEP$secondaryArchSuffix = $libVersionCompat
+	devel:libTKSTEP209$secondaryArchSuffix = $libVersionCompat
+	devel:libTKSTEPAttr$secondaryArchSuffix = $libVersionCompat
+	devel:libTKSTEPBase$secondaryArchSuffix = $libVersionCompat
+	devel:libTKSTL$secondaryArchSuffix = $libVersionCompat
+	devel:libTKTObj$secondaryArchSuffix = $libVersionCompat
+	devel:libTKTopAlgo$secondaryArchSuffix = $libVersionCompat
+	devel:libTKXMesh$secondaryArchSuffix = $libVersionCompat
+	devel:libTKXmlL$secondaryArchSuffix = $libVersionCompat
+	devel:libTKXSBase$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	opencascade$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libgomp$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:sed
+	"
+
+defineDebugInfoPackage opencascade$secondaryArchSuffix \
+	"$libDir"/libFWOSPlugin.so.$libVersion \
+	"$libDir"/libPTKernel.so.$libVersion \
+	"$libDir"/libTKBinL.so.$libVersion \
+	"$libDir"/libTKBO.so.$libVersion \
+	"$libDir"/libTKBool.so.$libVersion \
+	"$libDir"/libTKBRep.so.$libVersion \
+	"$libDir"/libTKCDF.so.$libVersion \
+	"$libDir"/libTKernel.so.$libVersion \
+	"$libDir"/libTKFeat.so.$libVersion \
+	"$libDir"/libTKFillet.so.$libVersion \
+	"$libDir"/libTKG2d.so.$libVersion \
+	"$libDir"/libTKG3d.so.$libVersion \
+	"$libDir"/libTKGeomAlgo.so.$libVersion \
+	"$libDir"/libTKGeomBase.so.$libVersion \
+	"$libDir"/libTKHLR.so.$libVersion \
+	"$libDir"/libTKIGES.so.$libVersion \
+	"$libDir"/libTKLCAF.so.$libVersion \
+	"$libDir"/libTKMath.so.$libVersion \
+	"$libDir"/libTKMesh.so.$libVersion \
+	"$libDir"/libTKOffset.so.$libVersion \
+	"$libDir"/libTKPLCAF.so.$libVersion \
+	"$libDir"/libTKPrim.so.$libVersion \
+	"$libDir"/libTKPShape.so.$libVersion \
+	"$libDir"/libTKShapeSchema.so.$libVersion \
+	"$libDir"/libTKShHealing.so.$libVersion \
+	"$libDir"/libTKStdLSchema.so.$libVersion \
+	"$libDir"/libTKSTEP.so.$libVersion \
+	"$libDir"/libTKSTEP209.so.$libVersion \
+	"$libDir"/libTKSTEPAttr.so.$libVersion \
+	"$libDir"/libTKSTEPBase.so.$libVersion \
+	"$libDir"/libTKSTL.so.$libVersion \
+	"$libDir"/libTKTObj.so.$libVersion \
+	"$libDir"/libTKTopAlgo.so.$libVersion \
+	"$libDir"/libTKXMesh.so.$libVersion \
+	"$libDir"/libTKXmlL.so.$libVersion \
+	"$libDir"/libTKXSBase.so.$libVersion
+
+BUILD()
+{
+	mkdir -p build && cd "$_"
+
+	cmake .. \
+		-DOCE_MULTITHREAD_LIBRARY=OPENMP \
+		-DOCE_DISABLE_X11=ON \
+		-DOCE_INSTALL_PREFIX=$prefix \
+		$cmakeDirArgs \
```

Comment:
This package does not use `GNUInstallDirs` sadly, but they do [provide][0] an equivalent to it, which would help you clean up the moving mess in `INSTALL()`

[0]: https://github.com/tpaviot/oce/blob/f82e77df1fdfad4a8810064d9109d4fec3419ea7/CMakeLists.txt#L856